test/build/preserve-volumes/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine
```

Comment:
I would prefer that we do some tests with Fedora, not always ALpine.

Reply:
I've had to remove completely.  Can't get Buildah to install on the version of Ubuntu that Travis uses and can't run it inside of a Docker container (Chris E's solution).  Will have to develop some "by hand" tests until we can get Buildah to install Ubuntu.

Reply:
Just a footnote, per @rhatdan I've put these test back, but will try to not run them if Buildah is not installed.